subsys/random/Kconfig

```diff
@@ -53,4 +53,41 @@ config XOROSHIRO_RANDOM_GENERATOR
 
 	  It is so named because it uses 128 bits of state.
 
-endchoice
+endchoice # RNG_GENERATOR_CHOICE
+
+#
+# Implied dependency on a cryptographically secure entropy source when
+# enabling CS generators. ENTROPY_HAS_DRIVER is the flag indicating the
+# CS entropy source.
+#
+config CSPRING_ENABLED
```

Comment:
Right now you don't seem to be using this option for anything more than the test(?)
I guess you wanted to ifdef sys_csrand* with it and have the choice below depend on it?